script.js

```diff
@@ -64,3 +64,41 @@ function getCounty(lat, lon) {
 // Test cases
 // getCounty(0, 0);
 // getCounty(47.487983, -121.723172);
+
+
+function popularity (trailArray) {
```

Comment:
Suggestion(I will still approve if you don't address this one comment):
Instead of putting trailArray as a parameter of your functions, since trailArray is a global variable, you can define the function with no param.
Function declaration:
`function popularity() {
...
}`
Function call:
`popularity()`

Reply:
I would like to leave this in because I want it to be as explicit as possible. Just so it is easy to tell what the function is modifying when it is called. But I do not feel strongly about it